brave/brave-core
# Adds contribution queue

Resolves https://github.com/brave/brave-browser/issues/6288

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
- enable rewards with short contribution interval
- claim grant (30 BAT)
- set AC amount to 20 BAT
- add verified publishers to AC table
- add verified recurring tips in this order (5, 10, 10, 10)
- wait for monthly contribution to trigger
- you should see 25 BAT goes through for monthly and 5 BAT from AC

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
